### ISSUE TITLE
Replace homepage with Calendar page

### ISSUE THREAD

**Issue by natea** [SmallsLIVE/smallslive#273]
@natea : I have replaced home page with calendar page. Please have a look here - http://smallslive-staging.herokuapp.com/

**Comment by natea** [comment]
@rajpaul we need to confirm with Spike that this one is done.

**Comment by rajpaul** [comment]
@natea Yes! Spike asked us to do some more changes in front page which I have been working on. I will ask Spike after once I am done with these changes.
